src/config/dark-sites.config

```diff
@@ -1145,3 +1145,5 @@ zt64.github.io
 ztdp.ca
 zunivers.zerator.com
 zyrenth.dev
+wuemeli.com
```

Comment:
All pages in this file must be sorted alphabeticaly - correct that and we can merge that.

Reply:
Sorted :)